test/sync/src/androidTest/kotlin/io/realm/test/mongodb/shared/EmailPasswordAuthTests.kt

```diff
@@ -181,55 +193,56 @@ class EmailPasswordAuthTests {
         // Only emails containing realm_tests_do_autoverify will be confirmed
         val email = "do_not_confirm@10gen.com"
         app.setAutomaticConfirmation(false)
+        app.setCustomConfirmation(true)
         try {
```

Comment:
You shouldn't need this `try`-`finally` if you are using `assertFailsWith`. Clean up after the assertion normally.

Reply:
Same as above, we need the finally to be able to reset the state if this test fails.